packages/app/components/Questions/AsyncQuestions/TAanswerQuestionModal.tsx

```diff
@@ -84,20 +89,15 @@ export function AnswerQuestionModal({
       >
         <Form.Item
           name="answerText"
-          rules={[{ required: true, message: "Please input your response." }]}
+          rules={[{ required: true, message: 'Please input your response.' }]}
```

Comment:
![image](https://github.com/ubco-db/office-hours/assets/66136750/a65fbb2f-031c-42f7-9abe-48a3e85afa1e)
The allowClear overlaps with scrollbar, I think we can remove it for now maybe?